conans/client/rest/client_routes.py

```diff
@@ -129,10 +129,6 @@ class ClientV2Router(ClientCommonRouter):
     def __init__(self, base_url):
         self.base_url = "{}/v2/".format(base_url)
 
-    def recipe_latest(self, ref, path):
```

Comment:
This is the duplicated method removed, please @lasote check

Reply:
Notice that this method is used in here https://github.com/conan-io/conan/blob/2e5b21e82618d471c16e8ad6b8872d80c03fe609/conans/client/rest/rest_client_v2.py#L298

Probably the usage is not covered by tests

Reply:
Ok, it is redefined down below. Didn't see it 👍